src/storage/StorageServer.h

```diff
@@ -47,6 +47,10 @@ class StorageServer final {
  private:
   std::unique_ptr<kvstore::KVStore> getStoreInstance();
 
+  std::unique_ptr<kvstore::KVEngine> getAdmintStoreInstance();
+
+  int32_t getAdmintStoreSeqId();
```

Comment:
typo ? `getAdminStoreInstance` and `getAdminStoreInstance` ?

Reply:
> typo ? `getAdminStoreInstance` and `getAdminStoreInstance` ?

what do you mean?